src/components/mofo-footer/mofo-footer.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+export default React.createClass({
+  propTypes: {
+    footerLinks: React.PropTypes.arrayOf(React.PropTypes.shape({
+      iconType: React.PropTypes.oneOf([`info`, `email`, `github`, `chat`, `twitter`, `facebook`, `cc-license`, `code-of-conduct`, `cookies`, `legal`, `privacy`, `donate`,]).isRequired,
+      link: React.PropTypes.string.isRequired,
+      text: React.PropTypes.string.isRequired,
+      className: React.PropTypes.string
```

Comment:
Also, if you're seeing exceptions for certain sites we should flag that with design as inconsistent with the rest of the footers on our other sites.